config/config.json

```diff
@@ -20,7 +20,9 @@
         "InviteSalt": "gxHVDcKUyP2y1eiyW8S8na1UYQAfq6J6",
         "PublicLinkSalt": "TO3pTyXIZzwHiwyZgGql7lM7DG3zeId4",
         "ResetSalt": "IPxFzSfnDFsNsRafZxz8NaYqFKhf9y2t",
-        "AnalyticsUrl": ""
+        "AnalyticsUrl": "",
+        "UseLocalStorage": false,
+        "StorageDirectory": ""
```

Comment:
maybe turn on and set to relative path of data/ for unit testing and local dev builds?  (0/5)  then add data/ to the gitignore